src/Engine/Renderer/Renderer.hpp

```diff
@@ -428,8 +428,8 @@ class RA_ENGINE_API Renderer
   private:
     // Qt has the nice idea to bind an fbo before giving you the opengl context,
     // this flag is used to save it (and render the final screen on it)
-    int m_qtPlz;
-    int m_qtViewport[4];
+    //  int m_qtPlz;
+    //  int m_qtViewport[4];
```

Comment:
Why not removing these lines and the comment above if these are not needed anymore?